src/modelinstance.cpp

```diff
@@ -389,12 +392,13 @@ ov::AnyMap ModelInstance::getRTInfo(std::vector<std::string> path) {
 
 ov::AnyMap ModelInstance::getRTInfo() {
     OV_LOGGER("model: {}, ov::Model::get_rt_info<ov::AnyMap>()", reinterpret_cast<void*>(model.get()));
+    SPDLOG_LOGGER_DEBUG(modelmanager_logger, "Getting RTINFO");
```

Comment:
We do know that we are getting rt_info in L394. If really needed I would consider replacing log from 397 to actually log specific values of rt info in trace level?